### DISCUSSION TITLE
Loading an edge client with a custom path

### DISCUSSION THREAD

**Discussion by overmode** [prisma/prisma#23779] [Q&A] [answered]
### Question

I am following [this post](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) to create a setup with two databases. I would like to use Accelerate for both databases. My calls will be done in an edge environment, so I want to use the edge client as shown in [here](https://www.prisma.io/docs/accelerate/getting-started#24-extend-your-prisma-client-instance-to-add-the-accelerate-extension).

For non-edge clients, you would save your clients in custm paths and write

```
import { PrismaClient as PrismaClient1 } from '../prisma/generated/client1'
import { PrismaClient as PrismaClient2 } from '../prisma/generated/client2'
```

How can you do that for edge clients ? The doc doesn't seem to allow custom paths, they only show

```
import { PrismaClient } from '@prisma/client/edge'
```

If I save a client under the edge folder, will it work ?

**Comment by nurul3101** [comment, marked as answer]
Hello @overmode 👋 

> If I save a client under the edge folder, will it work ?
Yes, it should work as expected.

You would import it the same way from the custom path as you are doing it in non-edge client, the only change would be the addition of `/edge`.

Example:
```ts
import { PrismaClient as PrismaClient1 } from '../prisma/generated/client1/edge'
```

I hope this helps.
